Manual/Classes/DerivingHandlers.lean

```diff
@@ -54,7 +54,7 @@ private def checkDerivable (expected : Array Name) : CommandElabM Unit := do
 end
 
 
-#eval checkDerivable #[``BEq, ``DecidableEq, ``Hashable, ``Inhabited, ``Nonempty, ``Ord, ``Repr, ``SizeOf, ``TypeName, ``LawfulBEq, ``ReflBEq]
+#eval checkDerivable #[``BEq, ``DecidableEq, ``Hashable, ``Inhabited, ``Nonempty, ``Ord, ``Repr, ``SizeOf, ``TypeName, ``LawfulBEq, ``ReflBEq, ``Plausible.Arbitrary]
```

Comment:
This should have actually gone under `hiddenDerivable` above. It's not part of Lean itself.